package.json

```
@@ -92,11 +92,11 @@
     "packages/perf-benchmarks"
   ],
   "engines": {
-    "node": ">=10.13.0 <11.0.0",
-    "yarn": ">=1.10.1"
+    "node": ">=10.16.3 <10.17.0",
```

Is the upper bound necessary? Can we loosen it up a little and use the caret range specifier?
```suggestion
    "node": "^10.16.3",
```

---

We want to limit the build to use the same minor version because there is nonzero risk in allowing some builds to use a higher minor version. We want to decide at various times which minor version upgrade(s) we want to accept for all teams at once.